### PR TITLE
chore(dns): example cloud dns example with dnssec

### PR DESCRIPTION
**Pull request by msampathkumar**
Add cloud dns example for DNSSEC

Reference: https://cloud.google.com/dns/docs/dnssec-config#gcloud_1

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@msampathkumar
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
dns_managed_zone_with_dnssec/main.tf
--- old/dns_managed_zone_with_dnssec/main.tf
+++ new/dns_managed_zone_with_dnssec/main.tf
@@ -21,7 +21,7 @@
   dns_name    = "myzone.example.com."
   description = "Example Signed Zone"
   dnssec_config {
-      state = "on"
+    state = "on"
   }
 }
 # [END dns_managed_zone_with_dnssec]
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

File: dns_managed_zone_with_dnssec/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Google Cloud Documentation: https://cloud.google.com/dns/docs/dnssec-config#creating
+# [START dns_managed_zone_with_dnssec]
+resource "google_dns_managed_zone" "example" {
+  name        = "example-zone-name"
+  dns_name    = "myzone.example.com."
```
Comment (betsy-lichtenberg) [inline dns_managed_zone_with_dnssec/main.tf:21]: In the logs, I am seeing ": Error 400: Please verify ownership of the 'myzone.example.com.' domain (or a parent) at http://www.google.com/webmasters/verification/ and try again, verifyManagedZoneDnsNameOwnership"

How have you worked around this in other samples? Can we just example.com?

Comment (msampathkumar) [reply dns_managed_zone_with_dnssec/main.tf:21]: Hi @betsy-lichtenberg - in the past, this TF samples worked fine with `myzone.example.com.` but not now! Let me test with `example.com.`

**Comment by comment-bot-dev** [comment]
@msampathkumar
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by msampathkumar** [comment]
Noticed the same error again. Using `example.com.` (a generally recognisable example domain name) also did not work!

ManagedZone: googleapi: Error 400: The domain 'example.com.' (or its child domains) may be reserved or registered already, prohibited by policy, or there may be no DNS name server shards remaining for the name. For more information see https://cloud.google.com/dns/quotas#nameserver_limits.,

**Comment by betsy-lichtenberg** [comment]
We're getting these errors:

```
Error 400: The domain 'example.com.' (or its child domains) may be reserved or registered already, prohibited by policy, or there may be no DNS name server shards remaining for the name. For more information see https://cloud.google.com/dns/quotas#nameserver_limits., managedZoneDnsNameNotAvailable
```

Maybe comment it out or skip testing for this sample?

```
Error 409: The resource 'entity.managedZone' named 'peering-zone-e3a6a086b7c33f1b' already exists, alreadyExists
```

Maybe use a randomizer for the name?

**Comment by comment-bot-dev** [comment]
@msampathkumar
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**